changelog/fixed-ensure-armv7a-core-is-halted-for-endian-check.md

```diff
@@ -0,0 +1 @@
+Fixed: ensure that the ARMv7A core is halted when querying it for endianness.
```

Comment:
You don't need to changelog this, you're fixing a bug that hasn't been released yet

Reply:
Alright, I've removed that.